_data/projects/gkv.yml

```diff
@@ -0,0 +1,10 @@
+name: Gkv
+desc: An open source, key value store for small, simple projects leveraging git as the storage engine
+site: https://github.com/ybur-yug/gkv
+tags:
+  - database
+  - ruby
+  - git
+upforgrabs:
+  name: Gkv
+  link: https://github.com/ybur-yug/gkv/issues
```

Comment:
This should be a URL to the filtered list of issues curated for new contributors.

I can't see any particular labels which might suit this, but I'd suggest `help wanted`, which maps to this URL: https://github.com/ybur-yug/gkv/labels/help+wanted